app/views/nonprofits/show.json.jbuilder

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+# License: AGPL-3.0-or-later WITH WTO-AP-3.0-or-later
+# Full license explanation at https://github.com/houdiniproject/houdini/blob/main/LICENSE
+json.(@nonprofit, 
+    :id, 
+    :name, 
+    :url, 
+    :tagline,
+    :phone,
+    :email,
+    :website,
+    :summary,
+    :categories,
+    :full_description,
+    :published,
+    :no_anon,
+    :timezone,
+    :brand_color,
+    :hide_activity_feed
+    :state_code, 
+    :state_code_slug, 
+    :city, 
+    :city_slug,
+    :slug,
+    :country,
```

Comment:
After adding the above comma, I'm getting `ActionView::Template::Error (undefined method `country' for #<Nonprofit:0x00007fda297f6ad8>)`. Removing the `:country` symbol fixes this. Looking at `db/schema.rb` it appears that there's no such field on the `nonprofits` model. No big deal for my use-case, but might be worth considering in the future.